Version not reported

c8344295940410ec482a92e7f19b72ca5277da3c causes version to not be properly reported:
```text
$ KWStyle.exe -version
Version: ..
```
This in turn cause SimpleITK superbuild to fail for me:
```text
3>------ Build started: Project: SimpleITK, Configuration: RelWithDebInfo x64 ------
3>Performing configure step for 'SimpleITK'
3>Not searching for unused variables given on the command line.
3>loading initial cache file C:/a/Srd/SimpleITK-build/SimpleITK-build/CMakeCacheInit.txt
3>-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19043.
3>-- Building SimpleITK version "2.1.0"
3>-- Found PythonLibs: C:/a/Srd/python-install/libs/python36.lib (found suitable exact version "3.6.7")
3>-- Could NOT find Tclsh (missing: TCL_TCLSH)
3>-- Processing json files...
3>-- Processing json files...done
3>-- Found PythonLibs: C:/a/Srd/python-install/libs/python36.lib (found version "3.6.7")
3>CMake Error at Utilities/KWStyle/KWStyle.cmake:38 (MATH):
3>  MATH cannot parse the expression: "..*10000 + ..*100 + ..": syntax error,
3>  unexpected exp_TIMES (3).
3>Call Stack (most recent call first):
3>  CMakeLists.txt:354 (include)
3>
3>
3>-- Configuring incomplete, errors occurred!
3>See also "C:/a/Srd/SimpleITK-build/SimpleITK-build/CMakeFiles/CMakeOutput.log".
3>See also "C:/a/Srd/SimpleITK-build/SimpleITK-build/CMakeFiles/CMakeError.log".
3>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for 'C:\a\Srd\SimpleITK-build\CMakeFiles\817968a52b2ffd472839f230f3f09102\SimpleITK-configure.rule;C:\a\Srd\SimpleITK-build\CMakeFiles\817968a52b2ffd472839f230f3f09102\SimpleITK-forcebuild.rule;C:\a\Srd\SimpleITK-build\CMakeFiles\817968a52b2ffd472839f230f3f09102\SimpleITK-build.rule;C:\a\Srd\SimpleITK-build\CMakeFiles\817968a52b2ffd472839f230f3f09102\SimpleITK-install.rule;C:\a\Srd\SimpleITK-build\CMakeFiles\c983afcc39041e378e2317f06f74f815\SimpleITK-complete.rule;C:\a\Srd\SimpleITK-build\CMakeFiles\97eeb19693a25f42de1f28fa1df36358\SimpleITK.rule' exited with code 1.
3>Done building project "SimpleITK.vcxproj" -- FAILED.
```
as part of Slicer build.

This was introduced as part of #92.

--- a/KWStyleConfigure.h.in
+++ b/KWStyleConfigure.h.in
@@ -11,7 +11,4 @@
      PURPOSE.  See the above copyright notices for more information.
 
 =========================================================================*/
-#define KWSTYLE_MAJOR_VERSION "@KWStyle_MAJOR_VERSION@"
-#define KWSTYLE_MINOR_VERSION "@KWStyle_MINOR_VERSION@"
-#define KWSTYLE_PATCH_VERSION "@KWStyle_PATCH_VERSION@"
-#define KWSTYLE_VERSION_STRING "@KWStyle_MAJOR_VERSION@.@KWStyle_MINOR_VERSION@.@KWStyle_PATCH_VERSION@"
+#define KWSTYLE_VERSION_STRING "@KWStyle_VERSION@"

